src/sonic-yang-models/yang-models/sonic-port.yang

```diff
@@ -138,6 +138,10 @@ module sonic-port{
 					type stypes:tpid_type;
 				}
 
+				leaf mux_cable {
```

Comment:
>mux_cable [](http://example.com/codeflow?start=9&length=9)

Just FYI: if you would like to accept true and True, you may use stypes:boolean_type

Reply:
I believe the requirement for this field is that it must be lowercase. Thanks though!